src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java

```diff
@@ -115,8 +116,10 @@ public Uni<AsyncResultSet> executeSchemaChange(SimpleStatement boundStatement) {
             cqlSessionCache
                 .getSession()
                 .executeAsync(
-                    boundStatement.setSerialConsistencyLevel(
-                        operationsConfig.queriesConfig().consistency().schemaChanges())));
+                    boundStatement
+                        .setIdempotent(true)
```

Comment:
Are Schema change (DDL) really idempotent? I guess they can be if (and only if) we use "CREATE TABLE ... IF NOT EXISTS" and other LWT-based queries. Otherwise successful but timed out queries will be retried and fail (like trying to add same column again)

Reply:
Agreed. All the DDL statements in JSON API have IF NOT EXISTS kind of check which makes them idempotent.